File: Courses/edX/HarvardX/CS50's Introduction to Computer Science/Week 6/pset6/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

# Prompt User Input
while True:
    Height = get_int("Height: ")
    if Height >= 1 and Height <= 8:
        break

# Print Pattern
for i in range(Height):
    print(" " * (Height - i - 1), end="")
    print("#" * (i + 1), end="")
    print("  ", end="")
    print("#" * (i + 1))

File: Courses/edX/HarvardX/CS50's Introduction to Computer Science/Week 1/pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    // prompt user for length of ladder
    int a = 0;
    do
    {
        a = get_int("Height: ");
    }
    while (a < 1 || a > 8);

    // making the ladder
    for (int i = 1; i <= a; i++)
    {
        // left spaces
        for (int j = 0; j < (a - i); j++)
        {
            printf(" ");
        }

        // left ladder
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }

        // middle gap
        printf("  ");

        // right ladder
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }

        // next line
        printf("\n");
    }
}